jest.stylelint.config.js

```diff
@@ -1,8 +1,15 @@
 module.exports = {
   runner: 'jest-runner-stylelint',
   displayName: 'stylelint',
-  moduleFileExtensions: ['css'],
-  modulePathIgnorePatterns: ['dist'],
-  testMatch: ['<rootDir>/**/*.css'],
+  moduleFileExtensions: ['css', 'js'],
+  modulePathIgnorePatterns: [
+    'dist',
+    '.spec.js',
+    '.visualspec.js',
+    '.visualroute.js',
+    '.story.js',
+    'packages/mc-http-server/public/',
+  ],
+  testMatch: ['<rootDir>/**/*.css', '<rootDir>/packages/**/*.js'],
```

Comment:
We want to css-in-js lint packages for now. The website can be enabled later.

Reply:
What is the reason for not enabling it now?

Reply:
That it gives some errors I would like to follow up on after this. Also cause the website ain't 100% my turf :)